.gitignore

```
@@ -139,3 +139,6 @@ dmypy.json
 
 # Ignore version module generated by setuptools_scm
 /*/_version.py
+
+# Submodules
+/docs/sphinx-docs-starter-pack
```

```suggestion
/docs/sphinx-docs-starter-pack

```

EOL at the end of the file :-)